exonum/src/blockchain/config.rs

```diff
@@ -241,6 +242,17 @@ impl StoredConfiguration {
             )));
         }
 
+        // Check majority count
+        let validators_count = config.validator_keys.len() as u16;
+        let byzantine_majority_count =
+            State::byzantine_majority_count(validators_count as usize) as u16;
+        if let Err(e) = validate_majority_count(
```

Comment:
I think `mar_err` will be more readable here.

Reply:
```rust
validate_majority_count(
            config.majority_count,
            validators_count,
            byzantine_majority_count,
        ).map_err(Json::custom)?;
Ok(config)
```